Tasks/CargoAuthenticateV0/_buildConfigs/Wif/Tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "cargo-authenticate-tests",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Cargo Authenticate V0 Task Tests",
+  "main": "L0.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
```

Comment:
Do we need the tests folder under wif ? whyd doesnt node 20 have it?

Reply:
I'm not sure if we need it - I'm curious to the if the pipeline complains. I see they are not included in the Task root, but are included in the generated files (for both wif and node20).